scripts/makepeds_psana

```diff
@@ -534,18 +592,20 @@ if [[ $LCLS2 -gt 0 ]]; then
 	        #revisit when we run into permission problems.
     	        for calibcycle in {0..4}; do
                     nextcycle=$(( calibcycle + 1 ))
-		    CMD="epix10ka_pedestals_calibration -d $EPIX10K -e $EXP -r $RUN --stepnum $calibcycle --stepmax $nextcycle -L INFO"
-		    if [[ $HOSTNAME =~ "drp-srcf" ]]; then
-   		        CMD=$CMD' -x /cds/data/drpsrcf/'$HUTCH'/'$EXP'/xtc'
-                    fi
+		    CMD="epix10ka_pedestals_calibration -d $EPIX10K -e $EXP -r $RUN --stepnum $calibcycle --stepmax $nextcycle -L INFO -x dir=$XTCDIR"
```

Comment:
Is this the syntax here?  -x dir=/path?  Because the old line 539 was just -x /path.

Reply:
This changed in a recent release when Mikhail also made  -x more consisten across scripts.
 -x DSNAMEX, --dsnamex=DSNAMEX
                        dataset string or xtc file name or extension after
                        exp=<exp>:run=<run> beginning with colon ":", default
                        = None